Prolific/Prolific/Model Builders/Round Builder/RoundBuilder.h

```diff
@@ -58,6 +58,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (RoundBuilder *)addSubmission:(Snippet *)snippet;
 
+- (RoundBuilder *)updateExistingSubmissionWithSubmission:(Snippet *)updatedSnippet;
```

Comment:
this name is kinda unclear. Maybe `updateExistingSubmissionWithSnippet` since the snippet is passed in?